### PR TITLE
disable cache per default on CI...... :( (still possible to use it via label force-build-cache. yeah no need to rebuild everything jetty-util, jetty-io, jetty-server. jetty-client, all jetty-ee9 and jetty-ee10  when you only touch jetty-ee10-servlet

### PR DESCRIPTION
**Pull request by olamy**
Signed-off-by: Olivier Lamy <olamy@apache.org>

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -140,7 +140,10 @@ def useBuildCache() {
     labelNoBuildCache = pullRequest.labels.contains("build-no-cache")
   }
   def noBuildCache = (env.BRANCH_NAME == 'jetty-12.0.x') || labelNoBuildCache;
-  return !noBuildCache;
+  if(pullRequest.labels.contains("force-build-cache")) {
```
Comment (gregw) [inline Jenkinsfile:143]: I like that build labels start with "build", so how about:
```suggestion
  if(pullRequest.labels.contains("build-use-cache")) {
```

**Review by gregw — APPROVED** [review]
other than a naming niggle....

**Comment by olamy** [comment]
still a draft. no rush nor motivation to merge that...

**Comment by olamy** [comment]
currently there is the option to use the label `build-no-cache` so no real need to disable totally.
it;s still speed up build of PRs.

**Pull request closed without merging**